src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -60,7 +61,12 @@ export class TestRunner extends NodejsEnvTestRunner {
     async run(opts: WorkerTestRunnerRunOpts): Promise<WorkerRunTestResult> {
         this._runOpts = opts;
 
-        const results = await super.run({ ...opts, ExecutionThreadCls: wrapExecutionThread(this._socket) });
+        const results = await super
+            .run({ ...opts, ExecutionThreadCls: wrapExecutionThread(this._socket) })
+            .catch((err: Error) => {
+                throw filterExtraWdioFrames(err);
+            });
```

Comment:
It is not wrapped in `runInStacktraceHooks`, so we need to filter wdio frames in CT manually